studio/pages/index.tsx

```diff
@@ -42,8 +43,14 @@ const Home: NextPage = () => {
     }
 
     app.onProjectDeleted(response)
-    ui.setNotification({ category: 'success', message: `Deleted ${project.name} successfully!` })
+    if (project.subscription_tier_prod_id === STRIPE_PRODUCT_IDS.FREE) {
+      mutateProfile({
+        ...profile,
+        total_free_projects_owned: profile.total_free_projects_owned - 1,
+      })
+    }
```

Comment:
you can pass `mutateProfile` as a param into `app.onProjectDeleted` for cleaner code